projects/1.18-fabric/assets/dark-utilities/darkutils/lang/zh_cn.json

```
@@ -1 +1,127 @@
-{}
\ No newline at end of file
+{
+  "__comment_file": "Dark Utilities 的中文本地化",
+  "__comment_creative_tabs": "创造模式标签",
+  "itemGroup.darkutils.creative_tab": "Dark Utilities",
+  "__comment_blocks": "方块名称",
+  "block.darkutils.blank_plate": "空白板",
+  "block.darkutils.vector_plate": "向量板",
+  "block.darkutils.vector_plate_fast": "快速向量板",
+  "block.darkutils.vector_plate_extreme": "极速向量板",
+  "block.darkutils.vector_plate_ultra": "超极速向量板",
+  "block.darkutils.damage_plate": "伤害板",
+  "block.darkutils.damage_plate_maim": "致残板",
+  "block.darkutils.damage_plate_player": "玩家伤害板",
+  "block.darkutils.flame_plate": "火焰板",
+  "block.darkutils.slowness_plate": "缓慢板",
+  "block.darkutils.fatigue_plate": "挖掘疲劳板",
+  "block.darkutils.darkness_plate": "昏暗板",
+  "block.darkutils.hunger_plate": "饥饿板",
+  "block.darkutils.weakness_plate": "虚弱板",
+  "block.darkutils.poison_plate": "中毒板",
+  "block.darkutils.wither_plate": "凋零板",
+  "block.darkutils.alert_plate": "发光板",
+  "block.darkutils.levitation_plate": "漂浮板",
+  "block.darkutils.misfortune_plate": "霉运板",
+  "block.darkutils.slowfall_plate": "缓降板",
+  "block.darkutils.omen_plate": "不祥之兆板",
+  "block.darkutils.frost_plate": "冰冻板",
+  "block.darkutils.filter_undead": "生物过滤器（亡灵）",
+  "block.darkutils.filter_animal": "生物过滤器（动物）",
+  "block.darkutils.filter_arthropod": "生物过滤器（节肢动物）",
+  "block.darkutils.filter_player": "生物过滤器（玩家）",
+  "block.darkutils.filter_slime": "生物过滤器（史莱姆）",
+  "block.darkutils.filter_hostile": "生物过滤器（敌对生物）",
+  "block.darkutils.filter_villager": "生物过滤器（村民）",
+  "block.darkutils.filter_fire_immune": "生物过滤器（火焰免疫）",
+  "block.darkutils.filter_child": "生物过滤器（幼体）",
+  "block.darkutils.filter_golem": "生物过滤器（傀儡）",
+  "block.darkutils.filter_pet": "生物过滤器（宠物）",
+  "block.darkutils.filter_water": "生物过滤器（水生）",
+  "block.darkutils.filter_named": "生物过滤器（已命名）",
+  "block.darkutils.filter_raider": "生物过滤器（袭击）",
+  "block.darkutils.filter_illager": "生物过滤器（灾厄村民）",
+  "block.darkutils.filter_freeze_immune": "生物过滤器（冰冻免疫）",
+  "block.darkutils.filter_equipment": "生物过滤器（盔甲）",
+  "block.darkutils.filter_passenger": "生物过滤器（乘客）",
+  "block.darkutils.redstone_randomizer": "红石随机器",
+  "block.darkutils.shielded_redstone": "单面红石块",
+  "__comment_items": "物品名称",
+  "item.darkutils.charm_sleep": "睡眠魔符",
+  "item.darkutils.charm_portal": "传送门魔符",
+  "item.darkutils.charm_warding": "守护魔符",
+  "item.darkutils.tome_enchanting": "附魔之书",
+  "item.darkutils.tome_shadows": "阴影之书",
+  "item.darkutils.tome_sga": "银河之书",
+  "item.darkutils.tome_illager": "灾厄村民之书",
+  "item.darkutils.tome_runelic": "Runelic 之书",
+  "item.darkutils.tome_pigpen": "猪舍之书",

```

```suggestion
  "item.darkutils.tome_runelic": "Runelic之书",
  "item.darkutils.tome_pigpen": "Pigpen之书",
```
这里是DarkHax的另外两个模组，Runelic和Pigpen，分别加入了一种符文字体（类似Wingdings）和猪圈密码
顺便1.16之后的版本普通情况下西文字符和中文字符之间不再需要空格